engine/runtime-fat-jar/src/main/java/module-info.java

```diff
@@ -6,6 +6,11 @@
   requires org.graalvm.polyglot;
   requires org.graalvm.truffle;
   requires static org.slf4j;
+  // ydoc-server
+  requires io.helidon.webclient;
+  requires io.helidon.webclient.websocket;
+  requires io.helidon.webserver;
+  requires io.helidon.webserver.websocket;
```

Comment:
Good solution for now, if everything works with that. But at the end, we must ensure that `runtime.jar` does not require helidon modules. Let's keep that in mind and create a separate ticket for that before merging.